### ISSUE TITLE
When used with jamine-rails, it gets stuck on window.callPhantom when specs are run through the web app

### ISSUE THREAD

**Issue by TheJefe** [shepmaster/jasmine-junitreporter-gem#1]
This works ok when run from the command line, however, when running jasmine tests by going to `/specs` in your locally hosted rails application, it gets hung up at the end of the first test suite with no errors shown.
## To reproduce:
1. enable tracing in JUnitReporter.js
2. Start your rails application
3. goto '/specs'
4. open up your browser dev tools to see whats going on

At this point you will see it gets to the point where it will write the results first test suite, and it just hangs on the `window.callPhantom` call.

**Comment by shepmaster** [comment]
Am I understanding correctly that you are attempting to use this gem outside of PhantomJS? If so, then it definitely won't work, although the failure mode of "it just hangs" is not very nice.

To get running, it sounds like you will want to use an ERB version of the `JUnitReporter.boot.js` file that pulls from the environment to know which reporters to use. There's a half-example of that in the README.

We should probably check to see if PhantomJS is available and raise an error instead to help point out what to do in that case.

**Comment by shepmaster** [comment]
From a previous project, my `JUnitReporter.boot.js.erb` looked like:

``` js
(function() {
  var reporter = new jasmine.JUnitReporter({
    outputDir: <%= (ENV.fetch('CI_REPORTS') { raise "ENV['CI_REPORTS'] must be set!" }).to_s.to_json %>
  });
  jasmine.getEnv().addReporter(reporter);
})();
```

**Comment by TheJefe** [comment]
Yes and no.  Yes, I was trying to run jasmine tests using the mounted test runner at `/specs`, but no, I did not expect it to use JUnitReporter.js or generate junit tests.

With some help, I came to a solution where we check if the userAgent for phantomjs before adding the reporter.  The solution looked like this, in our `JUnitReporter.boot.js`

``` js
(function() {
    var reporter = new jasmine.JUnitReporter({
        outputDir: 'test/reports/jasmine'
    });
     console.log(window.navigator.userAgent)
    if(window.navigator.userAgent.indexOf("PhantomJS") > -1){
        jasmine.getEnv().addReporter(reporter);
    }
})();
```
